market/mysterium/dto.go

```diff
@@ -24,6 +24,12 @@ type CreateIdentityRequest struct {
 	Identity string `json:"identity"`
 }
 
+// CreatePayoutInfoRequest represents JSON request for creating payout info request
+type CreatePayoutInfoRequest struct {
+	Identity   string `json:"identity"`
+	EthAddress string `json:"payout_eth_address"`
```

Comment:
Please be consistent for tequilapi variable names - it's camelCase

Reply:
tequilapi variables changed to camelCase. This example refers to discovery api variable